src/main/java/io/wisetime/idea/branch/BranchHelper.java

```diff
@@ -38,4 +47,12 @@ private void updateFrameTitle() {
     }
   }
 
+  private void updateProjectTitle(String projectTitle) {
+    if (project instanceof ProjectEx) {
+      ((ProjectEx) project).setProjectName(projectTitle);
```

Comment:
This updated window title on windows:
![image](https://user-images.githubusercontent.com/25821770/143196597-798459bc-7ba6-4599-a422-8c75085c0645.png)
And few other places on all OS:
![image](https://user-images.githubusercontent.com/25821770/143196841-4609c9e1-7275-4da5-9d79-a3dc77ae99bb.png)
I've been using it on my machine for few days, branch name in recent list is helpful

Reply:
nice!